src/assets/release-notes-1.18.0.json

```diff
@@ -0,0 +1,4645 @@
+{
+  "73363": {
+    "commit": "67647ed5a379ea72d7db7b5461e9802aa1debdbf",
+    "text": "Minimize AWS NLB health check timeout when externalTrafficPolicy set to Local",
+    "markdown": "Minimize AWS NLB health check timeout when externalTrafficPolicy set to Local ([#73363](https://github.com/kubernetes/kubernetes/pull/73363), [@kellycampbell](https://github.com/kellycampbell)) [SIG Cloud Provider]",
+    "author": "kellycampbell",
+    "author_url": "https://github.com/kellycampbell",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/73363",
+    "pr_number": 73363,
+    "areas": ["cloudprovider", "provider/aws"],
+    "kinds": ["bug"],
+    "sigs": ["cloud-provider"]
```

Comment:
The JSON needs to contain the release version:
```json
    "release_version": "1.18.0"
```
Interesting, we have this bug since the RC and I bet it relates to the release-notes tooling.